receiver/dotnetdiagnosticsreceiver/config.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dotnetdiagnosticsreceiver
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+	// Optional directory name to save binary blobs read from the socket. This can
+	// be useful for troubleshooting/debugging. If left blank or unset, no blobs are
+	// saved.
+	BlobDir string `mapstructure:"blob_dir"`
```

Comment:
Why is this necessarily? We can use the file exporter for this. Am I misunderstood?

Reply:
This is a debugging option where the raw binary data that this receiver reads from the Unix domain socket can be saved to disk. The intent is to have a way to understand what happened if this receiver ever is unable to parse the binary stream. I also used it to create test data. You can see the implementation here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2200/files#diff-cd099edb259acb2737a72d12ed2de73cb9d3298b4991f1a7ba32751495d804feR65

Reply:
But you can do the same with a FileExporter correct?

Reply:
I would not add premature debugging options. Let's start with the minimum we need and improve later.

Reply:
Ok. Sounds good.